src/hooks_test.rs

```diff
@@ -185,7 +185,7 @@ mod create_hook_files_tests {
             assert_eq!(true, make_executable);
             Ok(())
         };
-        let result = create_hook_files(write_file, root_dir, git_hooks);
+        let result = create_hook_files(write_file, root_dir, git_hooks, &vec![]);
         assert_eq!(result, Ok(()));
     }
```

Comment:
Could you also add a new test that includes at least one hook in the skip list so that we have coverage of the new behavior?

One option for the new test would be to essentially copy the above test, but add new match arm(s) on the file_name match within the `write_file` closure that map up to whichever files are provided in the skip list with a body that does something like `panic!` (essentially, the test will verify we never call `write_file` with any hook files that are supposed to be skipped).

It's a mildly gnarly test, so definitely let us know if you have any questions!